relay/chunk_manager.go

```diff
@@ -0,0 +1,174 @@
+package relay
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	v2 "github.com/Layr-Labs/eigenda/core/v2"
+	"github.com/Layr-Labs/eigenda/encoding"
+	"github.com/Layr-Labs/eigenda/encoding/rs"
+	"github.com/Layr-Labs/eigenda/relay/cache"
+	"github.com/Layr-Labs/eigenda/relay/chunkstore"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"sync"
+)
+
+type chunkManager struct {
+	ctx    context.Context
+	logger logging.Logger
+
+	// metadataCache is an LRU cache of blob metadata. Blobs that do not belong to one of the relay shards
```

Comment:
What is "relay shard"?

Reply:
I've updated this comment to omit the term "relay shard". For interpreting this, you can just interpret `relay shard` to mean `relay ID`.

```
	// metadataCache is an LRU cache of blob metadata. Each relay is authorized to serve data assigned to one or more
	// relay IDs. Blobs that do not belong to one of the relay IDs assigned to this server will not be in the cache.
```